### PR TITLE
detect/modbus/v3: remove unittests

### PR DESCRIPTION
**Pull request by TheKharleeci**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [ ] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

Previous PR: #6831

Describe changes:
- remove unittests that have been converted to Suricata-Verify

suricata-verify-pr: 648

### CODE REVIEW

File: src/detect-modbus.c [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 ANSSI
+ * Copyright (C) 2014-2022 ANSSI
```
Comment (victorjulien) [inline src/detect-modbus.c:2]: no need to update this here, as we're only removing code

**Review by victorjulien — CHANGES_REQUESTED** [review]
Minor change, see inline comment

File: src/detect-modbus.c (hunk 1 above)
Comment (TheKharleeci) [reply src/detect-modbus.c:2]: Ok. Noted. Thank you.

**Comment by TheKharleeci** [comment]
Followed By: #6867

**Pull request closed without merging**